_data/jobs.yml

```diff
@@ -320,17 +320,6 @@
   name: Operations Team Lead - Institute for Computational and Data Sciences (ICDS)
   posted: 2021-11-09
   url: https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/Operations-Lead_REQ_0000020695-1
-- expires: 2022-06-30
-  location: State College, Pennsylvania
-  name: System Engineering and Integration Lead - Institute for Computational and
-    Data Sciences (ICDS)
-  posted: 2021-11-09
-  url: https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/System-Engineering-and-Integration-Lead_REQ_0000020693
-- expires: 2022-06-30
-  location: State College, Pennsylvania
-  name: ICDS Technical Director - Institute for Computational and Data Sciences (ICDS)
-  posted: 2021-11-09
-  url: https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/ICDS-Technical-Director_REQ_0000020689
```

Comment:
Are you sure about this one? URL seems okay. Did someone ask to remove it?

https://psu.wd1.myworkdayjobs.com/en-US/PSU_Staff/job/University-Park-Campus/ICDS-Technical-Director_REQ_0000020689